client/js/render.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
```

Comment:
Any chance you could add a new line between `"use strict";` and what comes after in your files?

If this is just too annoying for you, nevermind then, not a big deal :)